UnitystationLauncher/Services/ServerService.cs

```diff
@@ -40,14 +39,14 @@ private async Task<IReadOnlyList<Server>> GetServerListAsync()
         {
             Refreshing = true;
 
-            var data = await _http.GetStringAsync(ApiUrls.ServerListUrl);
-            var serverData = JsonConvert.DeserializeObject<ServerList>(data)?.Servers;
+            string data = await _http.GetStringAsync(ApiUrls.ServerListUrl);
+            List<Server>? serverData = JsonConvert.DeserializeObject<ServerList>(data)?.Servers;
             Log.Information("Server list fetched");
 
-            var servers = new List<Server>();
+            List<Server> servers = new();
             if (serverData != null)
             {
-                foreach (var server in serverData)
+                foreach (Server server in serverData)
```

Comment:
can you validate here that the server has a valid ip/domain and skip them in the listing if not?

Reply:
Yeah I can do that later today.

Reply:
ServerIp should be validated for a valid IP or a valid domain name now. If it is neither it should get dropped and log a warning.